.gitignore

```diff
@@ -4,3 +4,4 @@
 *.sublime-workspace
 Pipfile.lock
 Pipfile
+sgc/
```

Comment:
Why are we ignoring the sgc folder? It's not pip installable is it? I think we need the sgc package to be part of this repo.

Reply:
good point. we'd ignored other insatallation folders but you're correct, they were pip installable, I think.